### DISCUSSION TITLE
Adding Aromatic proportion feature.

### DISCUSSION THREAD

**Discussion by msudhanshu10** [rdkit/rdkit#3682] [Ideas]
This molecular descriptor will add a feature in the Descriptor class- 'AromaticProportion'. This feature is not inbuilt in rdkit. We have to compute it manually. I came across this feature while I was doing computation cheminformatics analysis in Delaney's research paper which includes the calculation of solubility of molecules.

AromaticProportion= no of aromatic atoms in a molecule/ no of heavy atoms in a molecule.
no of aromatic atoms and heavy atoms in a molecule can be calculated separately in a molecule with existing methods.

**Comment by greglandrum** [comment]
Hi,
Are you asking how to implement this yourself or requesting that it be added to the core RDKit?
-greg
